legend-pure-code-compiled-core-relational/src/main/resources/core_relational/relational/tests/testRelationalExtension.pure

```diff
@@ -30,6 +31,12 @@ function <<test.Test>> meta::relational::metamodel::execute::tests::testExecuteI
    true;
 }
 
+function <<test.Test>> meta::relational::tests::typeInference::testTranslateDbType():Boolean[1]
```

Comment:
Can we add a comment/TODO to remind us that we should move this into the new db specific test structure

Reply:
Sure